.github/workflows/build.yml

```diff
@@ -64,8 +66,10 @@ jobs:
       - name: Check all tests and binaries compilation
         run: cargo check --workspace --tests --lib --bins
 
-      - name: Check no_std compilation
-        run: cargo check --workspace --lib --no-default-features
+      - name: Check no_std support and WASM compilation
+        env:
+          RUSTFLAGS: -C target-cpu=generic
+        run: cargo build --target wasm32-unknown-unknown --no-default-features
```

Comment:
Do you think it would be beneficial to still keep a second check for non-wasm `no-std` support, just in case?

Reply:
yes, good idea. will add.